src/sealed.rs

```diff
@@ -63,17 +38,11 @@ pub mod spsc {
         }
 
         #[cfg(has_atomics)]
-        unsafe fn load_acquire<C>(x: *const Self) -> Self
-        where
-            C: XCore,
-        {
-            if C::is_multi_core() {
-                (*(x as *const AtomicU8)).load(Ordering::Acquire)
-            } else {
-                let y = (*(x as *const AtomicU8)).load(Ordering::Relaxed); // read
-                atomic::compiler_fence(Ordering::Acquire); // ▼
-                y
-            }
+        unsafe fn load_acquire(x: *const Self) -> Self {
+            (*(x as *const AtomicU8)).load(Ordering::Acquire)
+            // let y = (*(x as *const AtomicU8)).load(Ordering::Relaxed); // read
```

Comment:
If multi-core is being removed, does it make sense keeping this, or will multi-core reappear later on?

Reply:
I was thinking if I should keep it or remove it, I have not favor towards either.